### PR TITLE
Create zh_CN.lang

### PR DESCRIPTION
**Pull request by DLslime**
update 1.10.2

### CODE REVIEW

**Comment by TheCyberBrick** [comment]
Hey, thanks for updating it! I'll check it out soon

File: src/main/resources/assets/thebetweenlands/lang/zh_CN.lang [hunk 1]
```diff
@@ -0,0 +1,866 @@
...
+command.generic.noplayer=你必须是个玩家
+
+command.blevent.success.on='%s'事件开始了
+command.blevent.success.off='%s'事件结束了
+command.blevent.success.alloff=所有事件已结束
+command.blevent.success.enable=启用事件
+command.blevent.success.disable=关闭事件
+
+command.blevent.usage=/blevent <切换，激活|on|off|list|enable|disable>
+command.blevent.usage.toggle=/blevent 切换，激活 <事件名称>
```
Comment (TartaricAcid) [inline src/main/resources/assets/thebetweenlands/lang/zh_CN.lang]: Do not translate the `toggle`

File: src/main/resources/assets/thebetweenlands/lang/zh_CN.lang [hunk 2]
```diff
@@ -0,0 +1,866 @@
...
+#item.thebetweenlands.record.lonely_fire.desc=Voog2 - Lonely Fire
+#item.thebetweenlands.record.16612.desc=...
+#item.thebetweenlands.record.beneath_a_green_sky.desc=Voog2 - Beneath A 绿色 Sky
+#item.thebetweenlands.record.onwards.desc=Voog2 - Onwards
+#item.thebetweenlands.record.dj_wights_mixtape.desc=Voog2 - Rave In A Cave
+#item.thebetweenlands.record.stuck_in_the_mud.desc=Voog2 - Stuck In The 泥淖
+#item.thebetweenlands.record.waterlogged.desc=Voog2 - Waterlogged
+#item.thebetweenlands.record.wandering_wisps.desc=Voog2 - 法杖e指环 Wisps
+#item.thebetweenlands.record.lonely_fire.desc=Voog2 - Lonely Fire
+#item.thebetweenlands.record.ancient.desc=Voog2 - Ancient
```
Comment (TartaricAcid) [inline src/main/resources/assets/thebetweenlands/lang/zh_CN.lang]: Delete all Chinese in there entries.

**Review by TartaricAcid — CHANGES_REQUESTED** [review]
Thanks for your translation, but some entries need to be correct.

File: src/main/resources/assets/thebetweenlands/lang/zh_CN.lang (hunk 1 above)
Comment (DLslime) [reply src/main/resources/assets/thebetweenlands/lang/zh_CN.lang]: Thanks for reminding me.

**Review by TartaricAcid — APPROVED** [review]

**Pull request closed without merging**